src/Jackett.Common/Definitions/matrix.yml

```
@@ -0,0 +1,175 @@
+---
+id: matrix
+name: Matrix
+description: "M.A.T.R.I.X is a HUNGARIAN Private Tracker for MOVIES / TV / GENERAL"
+language: hu-HU
+type: private
+encoding: UTF-8
+links:
+  - https://matrixworld.info/
+
+caps:
+  categorymappings:
+    - {id: 23, cat: Movies/Other, desc: "Animációk"}
+    - {id: 61, cat: Movies/Other, desc: "Cam/EN"}
+    - {id: 60, cat: Movies/Other, desc: "Cam/HU"}
+    - {id: 45, cat: Books/EBook, desc: "eBook/EN"}
+    - {id: 40, cat: Books/EBook, desc: "eBook/HU"}
+    - {id: 20, cat: Movies/DVD, desc: "Film/DVD9"}
+    - {id: 48, cat: Movies/HD, desc: "Film/EN/1080p"}
+    - {id: 32, cat: Movies/HD, desc: "Film/EN/720p"}
+    - {id: 38, cat: Movies/BluRay, desc: "Film/EN/Blu-ray"}
+    - {id: 27, cat: Movies/DVD, desc: "Film/EN/DVD-R"}
+    - {id: 24, cat: Movies/SD, desc: "Film/EN/SD"}
+    - {id: 34, cat: Movies/HD, desc: "Film/HU/1080p"}
+    - {id: 31, cat: Movies/HD, desc: "Film/HU/720p"}
+    - {id: 35, cat: Movies/BluRay, desc: "Film/HU/Blu-ray"}
+    - {id: 26, cat: Movies/DVD, desc: "Film/HU/DVD-R"}
+    - {id: 5, cat: Movies/SD, desc: "Film/HU/SD"}
+    - {id: 4, cat: PC/Games, desc: "Játékok/ISO"}
+    - {id: 39, cat: PC/Games, desc: "Játékok/Rip/Dox"}
+    - {id: 47, cat: Other, desc: "Klippek"}
+    - {id: 30, cat: Audio/Lossless, desc: "Lossless/EN"}
+    - {id: 29, cat: Audio/Lossless, desc: "Lossless/HU"}
+    - {id: 25, cat: Audio/MP3, desc: "MP3/EN"}
+    - {id: 6, cat: Audio/MP3, desc: "MP3/HU"}
+    - {id: 33, cat: PC, desc: "Program/egyéb"}
+    - {id: 1, cat: PC/ISO, desc: "Program/ISO"}
+    - {id: 36, cat: TV/HD, desc: "Sorozat/EN/HD"}
+    - {id: 49, cat: TV/SD, desc: "Sorozat/EN/SD"}
+    - {id: 28, cat: TV/HD, desc: "Sorozat/HU/HD"}
+    - {id: 7, cat: TV/SD, desc: "Sorozat/HU/SD"}
+    - {id: 9, cat: XXX, desc: "XXX"}
+    - {id: 44, cat: XXX/Other, desc: "XXX/HD"}
+    - {id: 43, cat: XXX/SD, desc: "XXX/SD"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    book-search: [q]
+    music-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Filter freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 0
+    options:
+      0: created
+      1: title
+      4: size
+      6: seeders
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
```

```suggestion
    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
  - name: info_activity
    type: info
    label: Account Inactivity
    default: "If you do not log in to the site for 6 months, the system will automatically delete it!"
```

It also says `At the moment, the auto-delete function is not active`, but I assume that means it will be in future, so might as well just add it.

---

No opinion. ;-)